src/dials/algorithms/symmetry/cosym/__init__.py

```diff
@@ -382,15 +390,32 @@ def _reindexing_ops(
         coord_ids = np.arange(n_datasets * n_sym_ops)
         dataset_ids = coord_ids % n_datasets
 
-        # choose a high density point as seed
-        X = coords
-        nbrs = NearestNeighbors(
-            n_neighbors=min(11, len(X)), algorithm="brute", metric="cosine"
-        ).fit(X)
-        distances, indices = nbrs.kneighbors(X)
-        average_distance = np.array([dist[1:].mean() for dist in distances])
-        i = average_distance.argmin()
-        xis = np.array([X[i]])
+        if (
+            self.using_reference
+        ):  # if reference, choose the reference with reindexing op xyz as seed
+            xis = None
+            sel = np.where(dataset_ids == n_datasets - 1)
+            xis = np.array([coords[sel][0]])
+            coordstr = ",".join(str(round(i, 4)) for i in xis[0])
+            logger.info(f"Coordinate of reference dataset with cb_op=x,y,z: {coordstr}")
+            # We no longer need the reference dataset, so remove it from the dataset ids
+            self.dataset_ids = self.dataset_ids.select(
+                self.dataset_ids != n_datasets - 1
+            )
+            n_datasets -= 1
+            # FIXME trim coords and coords_reduced?
+        else:
+            # choose a high density point as seed
+            X = coords
+            nbrs = NearestNeighbors(
+                n_neighbors=min(11, len(X)), algorithm="brute", metric="cosine"
+            ).fit(X)
+            distances, indices = nbrs.kneighbors(X)
+            average_distance = np.array([dist[1:].mean() for dist in distances])
+            i = average_distance.argmin()
+            xis = np.array([X[i]])
+            coordstr = ",".join(str(round(i, 4)) for i in xis[0])
+            logger.info(f"High density cluster seed coordinates: {coordstr}")
```

Comment:
```suggestion
            # We no longer need the reference dataset, so remove it from the dataset ids
            self.dataset_ids = self.dataset_ids.select(
                self.dataset_ids != n_datasets - 1
            )
            n_datasets -= 1
            # FIXME trim coords and coords_reduced?
        else:
            # choose a high density point as seed
            X = coords
            nbrs = NearestNeighbors(
                n_neighbors=min(11, len(X)), algorithm="brute", metric="cosine"
            ).fit(X)
            distances, indices = nbrs.kneighbors(X)
            average_distance = np.array([dist[1:].mean() for dist in distances])
            i = average_distance.argmin()
            xis = np.array([X[i]])
```

Reply:
This can be useful for debugging in trickier cases, as it's not usually obvious where the highest cos-angle density is, but I think the debug log is more appropriate, so I've moved it to `logger.debug`